radicle-httpd/src/api/json.rs

```diff
@@ -130,16 +134,11 @@ fn name_in_path(path: &str) -> &str {
     }
 }
 
-#[derive(Serialize)]
-struct Author {
-    id: PublicKey,
-}
-
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
 struct Comment {
     id: OpId,
-    author: Author,
+    author: Did,
```

Comment:
This prevents us from for eg. including an avatar and "name" field in the author object in the future.

Reply:
I guess `assignees` should be the same?

Reply:
Has been reverted